### PR TITLE
fix(ui): improve network payment form design

### PR DESCRIPTION
**Pull request by wa0x6e**
### Summary

Implement UI by the designer (https://www.figma.com/file/feaNugqoha00BXHFjkLbbd/Snapshot-%2F-Design?node-id=3738%3A47248&mode=dev)

### CODE REVIEW

File: src/views/PaymentNetwork.vue [hunk 1]
```diff
@@ -74,54 +74,72 @@ function formatFiatCurrency(amount: number): string {
...
-  <p class="text-md mb-4 leading-5">Pay to support your network on Snapshot</p>
+  <h1 class="leading-[44px] mb-[2px]">Network plan</h1>
+  <p class="mb-4 leading-[20px]">Pay to support your network on Snapshot</p>
 
   <form v-if="fxLoadStatus === 1" class="flex flex-col gap-4" @submit="pay">
     <fieldset>
-      <legend>
-        <h2 class="text-lg mb-2">Select your plan duration</h2>
+      <legend
+        class="text-[20px] mb-[12px] leading-none text-skin-heading font-semibold"
```
Comment (samuveth) [inline src/views/PaymentNetwork.vue:89]:
```suggestion
        class="text-md mb-[12px] leading-none text-skin-heading font-semibold"
```

Comment (samuveth) [inline src/views/PaymentNetwork.vue:89]:
```suggestion
        class="text-md mb-[12px] leading-none text-skin-heading font-semibold"
```
You can see text sizes in tailwind.config

File: src/views/PaymentNetwork.vue [hunk 2]
```diff
@@ -74,54 +74,72 @@ function formatFiatCurrency(amount: number): string {
...
-
 watch(paymentTx, () => {
   modalPostPaymentOpen.value = !!paymentTx.value;
 });
 </script>
 
 <template>
-  <h1>Network plan</h1>
-  <p class="text-md mb-4 leading-5">Pay to support your network on Snapshot</p>
+  <h1 class="leading-[44px] mb-[2px]">Network plan</h1>
```
Comment (samuveth) [inline src/views/PaymentNetwork.vue:83]:
```suggestion
  <h1 class="leading-[44px] mb-[2px]">Network plan</h1>
```
`text-2xl` should already be 44px line height and same font size as design

File: src/views/PaymentNetwork.vue [hunk 3]
```diff
@@ -74,54 +74,72 @@ function formatFiatCurrency(amount: number): string {
...
+              >{{ Math.floor(plan.discount) }}%</BasePill
+            >
+          </div>
           <label
-            class="flex gap-3 border border-skin-border hover:border-skin-text rounded-xl p-3 cursor-pointer"
+            class="flex relative bg-skin-bg gap-3 ring-1 ring-skin-border hover:ring-skin-text rounded-xl py-[12px] px-3 cursor-pointer"
             :class="{
-              '!border-skin-primary': planId === data.plan
+              '!ring-skin-primary': planId === data.plan,
+              'ring-[#384aff]/0': plan.discount
```
Comment (samuveth) [inline src/views/PaymentNetwork.vue]: I think you can simply do `ring-none` or something like that?

File: src/views/PaymentNetwork.vue [hunk 4]
```diff
@@ -74,54 +74,72 @@ function formatFiatCurrency(amount: number): string {
...
-            <div class="flex-grow">
-              <b class="text-skin-heading">{{ plan.label }}</b>
-              <BasePill v-if="plan.discount" class="ml-2 py-1 !bg-skin-primary"
-                >-{{ Math.floor(plan.discount) }}%</BasePill
-              >
-              <small class="block text-skin-text">
-                {{ formatFiatCurrency(computePlanFiatPrice(plan, BASE_UNIT)) }}
-                per month
-              </small>
+            <div class="flex-grow leading-none gap-[2px] flex flex-col">
```
Comment (samuveth) [inline src/views/PaymentNetwork.vue:118]: What does leading-none do here?

File: src/views/PaymentNetwork.vue [hunk 5]
```diff
@@ -131,51 +149,67 @@ watch(paymentTx, () => {
...
-              <small class="leading-none">{{ currency.code }}</small>
-            </div>
+            <AvatarToken :address="currency.address[1]" size="18" />
+
+            <small class="leading-none text-skin-heading text-base">{{
+              currency.code
+            }}</small>
+            <span
+              v-if="currencyId === data.currency"
+              :class="['absolute right-0 pr-3']"
```
Comment (samuveth) [inline src/views/PaymentNetwork.vue:165]:
```suggestion
              class="absolute right-0 pr-3"
```

File: src/views/PaymentNetwork.vue [hunk 6]
```diff
@@ -131,51 +149,67 @@ watch(paymentTx, () => {
...
+      hint="By proceeding, I understand that completing the network request form is mandatory after payment."
+      @update:model-value="setData('termsAccepted', $event as boolean)"
     >
-      Pay
-      {{ formatCryptoCurrency(amount, CURRENCIES[data.currency]) }} for
-      {{ PLANS[data.plan].label }}
-    </TuneButton>
+    </TuneCheckbox>
+
+    <hr />
```
Comment (samuveth) [inline src/views/PaymentNetwork.vue:182]: Should use flexbox instead of new line

File: src/views/PaymentNetwork.vue [hunk 7]
```diff
@@ -74,54 +74,72 @@ function formatFiatCurrency(amount: number): string {
...
+        >
+          <div v-if="plan.discount" class="p-2 text-center text-skin-primary">
+            Save
+            <BasePill class="py-1 !bg-skin-primary"
+              >{{ Math.floor(plan.discount) }}%</BasePill
+            >
+          </div>
           <label
-            class="flex gap-3 border border-skin-border hover:border-skin-text rounded-xl p-3 cursor-pointer"
+            class="flex relative bg-skin-bg gap-3 ring-1 ring-skin-border hover:ring-skin-text rounded-xl py-[12px] px-3 cursor-pointer"
```
Comment (samuveth) [inline src/views/PaymentNetwork.vue:111]: Can we move remove the hover effect here and on currency selector?

File: src/views/PaymentNetwork.vue [hunk 8]
```diff
@@ -131,51 +149,67 @@ watch(paymentTx, () => {
...
+              computePrice(
+                data.currency,
+                PLANS[data.plan].unit,
+                PLANS[data.plan].discount
+              ).toLocaleString()
+            }}
+          </b>
+        </div>
+      </div>
+      <TuneButton
```
Comment (samuveth) [inline src/views/PaymentNetwork.vue:203]: Can we increase the width of the payment button?

**Review by samuveth — COMMENTED** [review]
Looks good, left some comments

File: src/views/PaymentNetwork.vue (hunk 4 above)
Comment (wa0x6e) [reply src/views/PaymentNetwork.vue:118]: Removing the default line-height

File: src/views/PaymentNetwork.vue (hunk 6 above)
Comment (wa0x6e) [reply src/views/PaymentNetwork.vue:182]: This is not a line return, but a line element

File: src/views/PaymentNetwork.vue (hunk 4 above)
Comment (samuveth) [reply src/views/PaymentNetwork.vue:118]: Should be one line below I think? Two lines below you add `leading-none` again

File: src/views/PaymentNetwork.vue (hunk 6 above)
Comment (samuveth) [reply src/views/PaymentNetwork.vue:182]: Oh :D

File: src/views/PaymentNetwork.vue (hunk 2 above)
Comment (wa0x6e) [reply src/views/PaymentNetwork.vue:83]: `text-2xl` line-height is 50px

File: src/views/PaymentNetwork.vue (hunk 8 above)
Comment (wa0x6e) [reply src/views/PaymentNetwork.vue:203]: Added `px-5`

File: src/views/PaymentNetwork.vue (hunk 7 above)
Comment (wa0x6e) [reply src/views/PaymentNetwork.vue:111]: You mean the border color change on hover ?

Comment (samuveth) [reply src/views/PaymentNetwork.vue:111]: Just no hover effect

Comment (wa0x6e) [reply src/views/PaymentNetwork.vue:111]: No hover effect on a clickable element ?

Seems inconsistent with other similar component, such as Spaces strategies/voting type selection

**Review by samuveth — APPROVED** [review]
tACK

**Comment by wa0x6e** [comment]
Wait, keyboard navigation seems broken since the removal of the radio boxes

File: src/views/PaymentNetwork.vue (hunk 7 above)
Comment (samuveth) [reply src/views/PaymentNetwork.vue:111]: Ok

**Pull request merged**